tests/lib/utils/ember-test.js

```diff
@@ -134,7 +134,7 @@ describe('isEmberCoreModule', () => {
     expect(emberUtils.isEmberCoreModule(context, node, 'Controller')).toBeTruthy();
   });
 
-  it('should check if current file is a route', () => {
+  it('should check if current file is a route (custom)', () => {
```

Comment:
why the suffix?

Reply:
To avoid having a duplicate `it` title.

https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-identical-title.md